legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/resources/ReviewsOnlyResource.java

```diff
@@ -60,6 +60,8 @@ public ReviewsOnlyResource(ReviewApi reviewApi)
                                    @QueryParam("authoredByMe")
                                    @DefaultValue("true")
                                    @ApiParam("Only include reviews authored/created by me if true") boolean authoredByMe,
+                                   @QueryParam("labels")
+                                   @ApiParam("Only include reviews with the given labels") Set<String> labels,
```

Comment:
Clarify the doc as to whether it's any or all of the labels we're searching for.